detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsLambdaDetector.ts

```diff
@@ -18,20 +18,20 @@ import {
   Detector,
   Resource,
   ResourceDetectionConfig,
-} from '@opentelemetry/resources';
+} from "@opentelemetry/resources";
```

Comment:
seems you should tell your editor to leave ' untouched :o)

Reply:
Seems to me it should be set at the repo level :D. But fair, will do